### DISCUSSION TITLE
Is it possible to create repositories using Github Apps / OAuth Apps?

### DISCUSSION THREAD

**Discussion by basantech89** [community/community#40654] [Repositories]
### Select Topic Area

Product Feedback

### Body

I'm building a CLI application in NodeJS, and a part of it needs to create a repository on behalf of users in their GitHub account. I'd like to use the browser for authentication in case it's available in the user's environment. I looked at articles but I'm confused if I should use a Github app or an OAuth app or something else.

**Comment by ossanna16** [comment]
Hi there @basantech89 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
